frontend/src/components/courses/UnitSidebar.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from "react";
+import {
+  Box,
+  Button,
+  Divider,
+  Drawer,
+  List,
+  ListItemButton,
+  ListItemText,
+  useTheme,
+} from "@mui/material";
+import ArrowBackIosIcon from "@mui/icons-material/ArrowBackIos";
+import { CourseUnit } from "../../types/CourseTypes";
+import { neutral } from "../../theme/palette";
+
+interface UnitSideBarProps {
+  courseUnits: CourseUnit[];
+  handleClose: () => void;
+  open: boolean;
+}
+
+export default function UnitSidebar(props: UnitSideBarProps) {
+  const theme = useTheme();
+  const { courseUnits, handleClose, open } = props;
+
+  const [selectedIndex, setSelectedIndex] = useState(0);
+
+  const handleListItemClick = (
+    event: React.MouseEvent<HTMLDivElement, MouseEvent>,
+    index: number,
+  ) => {
+    setSelectedIndex(index);
+  };
+
+  return (
+    <Drawer
+      sx={{
+        width: "20%",
+        flexShrink: 0,
+        "& .MuiDrawer-paper": {
+          width: "20%",
+          boxSizing: "border-box",
+        },
+        display: open ? "block" : "none",
+      }}
+      variant="persistent"
+      anchor="left"
+      open={open}
+    >
+      <Box height="100%" sx={{ backgroundColor: theme.palette.neutral.light }}>
+        <Box
+          height="59px"
+          display="flex"
+          p="12px"
```

Comment:
`paddingLeft` should be 20px here:
![image](https://github.com/user-attachments/assets/db4524e4-0e09-4430-8af6-49fb07d55730)